airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.json

```diff
@@ -15,16 +15,77 @@
       },
       "credentials": {
         "type": "object",
```

Comment:
we should set the `title` and `description` properties for this `credentials` block. You can use instructions here https://github.com/airbytehq/airbyte/blob/master/docs/connector-development/connector-specification-reference.md for testing it out.